components/kyma-environment-broker/internal/provisioner/graphqlizer.go

```diff
@@ -139,6 +141,11 @@ func (g *Graphqlizer) UpgradeClusterConfigToGraphQL(in gqlschema.UpgradeClusterI
 }
 
 func (g *Graphqlizer) KymaConfigToGraphQL(in gqlschema.KymaConfigInput) (string, error) {
+	for _, c := range in.Components {
```

Comment:
personally I do not like when method is changing the input param when the method name does not say that. 

Maybe you better will to add just a quote function to tmpl engine in https://github.com/kyma-incubator/compass/pull/1132/files#diff-3b004e25742ce2ba4a362869881893e9R193
```go
fm["strQuote"] = strconv.Quote
```

then you can use `{{ .Value | strQuote }}`


working example: https://play.golang.org/p/Pu58JEhFCCg